onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -81,6 +81,13 @@ def graphs(self):
                             graph_queue.append(g)
         return self.all_graphs
 
+    def get_graphs_output_name(self):
```

Comment:
nit: how about get_graphs_output_names or get_graphs_output

Reply:
changed to get_graphs_output_names()